<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>card</class>
 <widget class="QMainWindow" name="card">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>card</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-image= url(&quot;C:/Users/Daniel/Documents/build-21CardGame-Desktop_Qt_6_5_0_MinGW_64_bit-Debug/img/background.png&quot;);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="push">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color: white;
border-radius: 10px;
background-color:#F34B37;
}</string>
    </property>
    <property name="text">
     <string>Push</string>
    </property>
   </widget>
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>561</width>
      <height>311</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../build-21CardGame-Desktop_Qt_6_5_0_MinGW_64_bit-Debug/img/background.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="rivalcard_1">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>61</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background-color: yellow;
}</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="rivalcard_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>61</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background-color: yellow;
}</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="rivalcard_3">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>61</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background-color: yellow;
}</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="mycard_1">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>190</y>
      <width>61</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background-color: yellow;
}</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="mycard_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>190</y>
      <width>61</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background-color: yellow;
}</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="mycard_3">
    <property name="geometry">
     <rect>
      <x>321</x>
      <y>190</y>
      <width>61</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background-color: yellow;
}</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="enemy1">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>15</y>
      <width>51</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../build-21CardGame-Desktop_Qt_6_5_0_MinGW_64_bit-Debug/img/top.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="enemy2">
    <property name="geometry">
     <rect>
      <x>245</x>
      <y>15</y>
      <width>51</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../build-21CardGame-Desktop_Qt_6_5_0_MinGW_64_bit-Debug/img/top.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="enemy3">
    <property name="geometry">
     <rect>
      <x>325</x>
      <y>15</y>
      <width>51</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../build-21CardGame-Desktop_Qt_6_5_0_MinGW_64_bit-Debug/img/top.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="card1">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>195</y>
      <width>51</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../build-21CardGame-Desktop_Qt_6_5_0_MinGW_64_bit-Debug/img/top.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="card2">
    <property name="geometry">
     <rect>
      <x>245</x>
      <y>196</y>
      <width>51</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../build-21CardGame-Desktop_Qt_6_5_0_MinGW_64_bit-Debug/img/top.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="card3">
    <property name="geometry">
     <rect>
      <x>326</x>
      <y>196</y>
      <width>51</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../build-21CardGame-Desktop_Qt_6_5_0_MinGW_64_bit-Debug/img/top.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="card_enemy_6">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>190</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../build-21CardGame-Desktop_Qt_6_5_0_MinGW_64_bit-Debug/img/Deniche.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="card_enemy_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../build-21CardGame-Desktop_Qt_6_5_0_MinGW_64_bit-Debug/img/Red.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="nombreRival_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>72</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>7</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
color:white;
background-color:#353939;
}</string>
    </property>
    <property name="text">
     <string>RED TRAINER</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="nombreRival_3">
    <property name="geometry">
     <rect>
      <x>469</x>
      <y>260</y>
      <width>73</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>9</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
color:white;
background-color:#353939;
}</string>
    </property>
    <property name="text">
     <string>DENNISEH</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="youPoint">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>142</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>9</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
color:white;
background-color:#353939;
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="nombreRival_5">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>120</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>9</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
color:white;
background-color:transparent;
border-color: transparent
}</string>
    </property>
    <property name="text">
     <string>YOU:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="nombreRival_6">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>120</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>9</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
color:white;
background-color:transparent;
border-color: transparent
}</string>
    </property>
    <property name="text">
     <string>RIVAL:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="rivalPoint">
    <property name="geometry">
     <rect>
      <x>338</x>
      <y>142</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>9</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
color:white;
background-color:#353939;
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="upcard">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color: white;
border-radius: 10px;
background-color:#164BBD;
}</string>
    </property>
    <property name="text">
     <string>Card +1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="play">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>110</y>
      <width>251</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color: white;
border-radius: 10px;
background-color:#DE2C31;
}</string>
    </property>
    <property name="text">
     <string>PLAY</string>
    </property>
   </widget>
   <widget class="QLabel" name="initial">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>561</width>
      <height>311</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../build-21CardGame-Desktop_Qt_6_5_0_MinGW_64_bit-Debug/img/background.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="exit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>190</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color: white;
border-radius: 10px;
background-color:#182DA3;
}</string>
    </property>
    <property name="text">
     <string>EXIT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="try_2">
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>190</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color: white;
border-radius: 10px;
background-color:Black;
}</string>
    </property>
    <property name="text">
     <string>CONTINUE</string>
    </property>
   </widget>
   <widget class="QLabel" name="finalmessage">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>120</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
color:white;
background-color: black;

}</string>
    </property>
    <property name="text">
     <string>DRAW</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>background</zorder>
   <zorder>push</zorder>
   <zorder>rivalcard_1</zorder>
   <zorder>rivalcard_2</zorder>
   <zorder>rivalcard_3</zorder>
   <zorder>mycard_1</zorder>
   <zorder>mycard_2</zorder>
   <zorder>mycard_3</zorder>
   <zorder>enemy1</zorder>
   <zorder>enemy2</zorder>
   <zorder>enemy3</zorder>
   <zorder>card1</zorder>
   <zorder>card2</zorder>
   <zorder>card3</zorder>
   <zorder>card_enemy_6</zorder>
   <zorder>card_enemy_4</zorder>
   <zorder>nombreRival_2</zorder>
   <zorder>nombreRival_3</zorder>
   <zorder>youPoint</zorder>
   <zorder>nombreRival_5</zorder>
   <zorder>nombreRival_6</zorder>
   <zorder>rivalPoint</zorder>
   <zorder>upcard</zorder>
   <zorder>initial</zorder>
   <zorder>play</zorder>
   <zorder>exit</zorder>
   <zorder>try_2</zorder>
   <zorder>finalmessage</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
